core/src/test/java/org/apache/calcite/jdbc/CalciteRemoteDriverTest.java

```diff
@@ -271,7 +271,7 @@ protected static Connection getRemoteConnection() throws SQLException {
   @Test public void testRemoteTypeInfo() throws Exception {
     CalciteAssert.hr().with(REMOTE_CONNECTION_FACTORY)
         .metaData(GET_TYPEINFO)
-        .returns(CalciteAssert.checkResultCount(is(45)));
+        .returns(CalciteAssert.checkResultCount(is(46)));
   }
```

Comment:
Why change this number?

Reply:
As LAMBDA Type is added, count is increased by 1 :)

Reply:
Got it. Thank you~~